server/server.go

```diff
@@ -469,6 +513,7 @@ func Serve(allowlist []string, sessionKey string, isdev bool) {
 	http.HandleFunc("/logout", handler.LogoutRoute)
 	http.HandleFunc("/login", handler.LoginRoute)
 	http.HandleFunc("/register", handler.RegisterRoute)
+	http.HandleFunc("/post/delete/", handler.DeletePostRoute)
```

Comment:
Odd to not have an id in a REST api like this IMO.

Like:

* POST /foo - creates foo
* GET /foo/<id> - gets one foo by id
* POST /foo/<id>/frob - frobs the foo with id <id>

and so on. Normally the route contains the canonical ID?

Reply:
iirc variables in routes are a deficiency of golang's default router / handler functions. the route *is* actually `POST /post/delete/<id>`

Reply:
might revise this later; appreciate the shout re it :)